react/src/components/awards/SprkAward.js

```diff
@@ -78,38 +82,57 @@ const SprkAward = (props) => {
         })}
       </SprkStack>
 
-      <SprkToggle
-        title={disclaimerTitle}
-        analyticsString={disclaimerAnalytics}
-        additionalClasses="sprk-o-Stack__item"
-      >
-        <p className="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
-          {disclaimerText}
-        </p>
-      </SprkToggle>
+      {(disclaimerTitle || disclaimerText) && (
+        <SprkToggle
+          title={disclaimerTitle}
+          analyticsString={disclaimerAnalytics}
+          additionalClasses="sprk-o-Stack__item"
+        >
+          <p className="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
+            {disclaimerText}
+          </p>
+        </SprkToggle>
+      )}
     </SprkStack>
   );
 };
 
 SprkAward.defaultProps = {
   splitAt: 'small',
   itemSpacing: 'medium',
-  additionalClasses: '',
-  idString: '',
-  heading: '',
-  disclaimerText: '',
-  disclaimerAnalytics: '',
-  disclaimerTitle: '',
 };
 
 SprkAward.propTypes = {
   /** Determines the spacing between the items. */
-  itemSpacing: PropTypes.string,
+  itemSpacing: PropTypes.oneOf([
+    'tiny',
+    'small',
+    'medium',
+    'large',
+    'huge',
+    'misc-a',
+    'misc-b',
+    'misc-c',
+    'misc-d',
+    'miscA',
```

Comment:
I'm wondering if we deprecated one of the sets of values here. I don't think we want both misc-b and miscB for example.

Reply:
i'll check

Reply:
It doesn't look like anything is deprecated currently. These are all still functional.

```
'sprk-o-Stack--misc-a': itemSpacing === 'miscA' || itemSpacing === 'misc-a',
'sprk-o-Stack--misc-b': itemSpacing === 'miscB' || itemSpacing === 'misc-b',
'sprk-o-Stack--misc-c': itemSpacing === 'miscC' || itemSpacing === 'misc-c',
'sprk-o-Stack--misc-d': itemSpacing === 'miscD' || itemSpacing === 'misc-d',
```

Reply:
Thanks for checking! Looks like the hyphenated values should be deprecated since our naming format for props is camelCase. Lets get an issue in to deprecate those in stack and not reference them here.

Reply:
@afebbraro I created a new Issue to deprecate these prop values in `SprkStack`. I marked them as deprecated in `SprkAward` since they may currently be in use in production. Let me know what you think.

Reply:
Good call on marking them deprecated here.